vendor/re2c.wake

```diff
@@ -16,10 +16,25 @@
 package build_wake
 from wake import _
 
-# Require re2c >= 1.0 for flag support
-def re2cOk Unit =
-    require Some x, Some y, Some z, _ =
-        makeExecPlan (which "re2c", "--version", Nil) Nil
+def re2cPath =
+    require Some path =
+        getenv "PATH"
+
+    def re2c =
+        whichIn path "re2c"
+
+    if matches `/.*` re2c then
+        Some re2c
+    else
+        None
```

Comment:
Use `re2c == ""` instead, so in the reverse of this order. That way you don't use a regex.

Reply:
Yeah I think that would be the best thing. Let's just do the `==~ "re2c"` thing for now